docs/source/index.rst

```diff
@@ -85,10 +85,17 @@ Composer is part of the broader Machine Learning community, and we welcome any c
    trainer/checkpointing.rst
    trainer/logging.rst
    trainer/callbacks.rst
-   trainer/distributed_training.rst
-   trainer/numerics.md
    trainer/performance.rst
 
+.. toctree::
+   :hidden:
+   :maxdepth: 1
+   :caption: Notes
+
+   trainer/distributed_training.rst
```

Comment:
Would it be possible to move these files to the "notes" folder to match the hierarchy in the index file?